src/utils/checkInit.js

```diff
@@ -0,0 +1,5 @@
+export default function checkInit(component) {
```

Comment:
`tipsi-stripe` is not a component. It's a module. Also, you've checked the property, not for the component, but for the instance of the module.
```js
export default function checkInit(instance) {
```